### DISCUSSION TITLE
Path with extension

### DISCUSSION THREAD

**Discussion by lostb1t** [salvo-rs/salvo#357] [Q&A]
Im truing to route an html file to an handle but it's not working. Get an 404 on http:/0.0.0.0:80/web/index.html

```
        .push(
            Router::new()
                .path("/web/index.html")
                .handle(redirect),
        )
```

Also can I have multiple paths for a router? 

        .push(
            Router::new()
                .path("/web/index.html")
                 .path("/web/another.html")
                .handle(redirect),
        )

**Comment by chrislearn** [comment]
Please provide a minimal code repository that can reproduce this issue.
